packages/react-codemirror/src/richClipboardCopier.ts

```diff
@@ -0,0 +1,74 @@
+import { EditorView } from '@codemirror/view';
+
+export function getCSSStyleForClass(className: string): string {
```

Comment:
I'm wondering, could we use [getComputedStyle](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) here instead? This way we don't have to deal with the differences between class application vs inline styles etc